docs/pages/blog/mui-x-v6-alpha-zero.md

```diff
@@ -0,0 +1,96 @@
+---
+title: MUI X v6.0.0-alpha.0
+description: We’re kicking off the development of the next version.
```

Comment:
How could we describe this announcement in the title and description to make it more exciting? Why should I, as an MUI X user and casual reader of the blog, be excited about this alpha zero? What's in it for me?

Reply:
There's the issue that the alpha.0 package per se, doesn't bring anything new.
But I guess we can use v6 in general for this.

```suggestion
title: MUI X v6 Alpha
description: Improved customization and usability. We’re kicking off the next major update for the advanced components.
```

Reply:
@samuelsycamore, I'm commiting this one, feel free to suggest anything else.